dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/GetOrderBookInfoSpec.scala

```diff
@@ -1,23 +1,40 @@
 package com.wavesplatform.it.matcher.api.http
 
 import com.typesafe.config.{Config, ConfigFactory}
-import com.wavesplatform.dex.api.http.entities.HttpMatchingRules
+import com.wavesplatform.dex.api.http.entities.{HttpMatchingRules, HttpOrderRestrictions}
 import com.wavesplatform.dex.domain.asset.AssetPair
 import com.wavesplatform.dex.it.api.RawHttpChecks
 import com.wavesplatform.it.MatcherSuiteBase
 import org.scalatest.prop.TableDrivenPropertyChecks
 
 class GetOrderBookInfoSpec extends MatcherSuiteBase with TableDrivenPropertyChecks with RawHttpChecks {
 
+  val minAmount = 0.1
+  val maxAmount = 100000000
+  val stepAmount = 0.1
+  val minPrice = 0.0001
+  val maxPrice = 1000
+  val stepPrice = 0.001
+
```

Comment:
You can create `expectedOrderRestrictions : HttpOrderRestrictions` here and import it in `dexInitialSuiteConfig`:
```scala
override protected def dexInitialSuiteConfig: Config = {
  import expectedOrderRestrictions._
  // ...
}
```